dev-lua/luarocks/luarocks-3.11.1.recipe

```diff
@@ -27,7 +27,7 @@ PROVIDES="
 REQUIRES="
 	haiku$secondaryArchSuffix
 	lib:liblua$secondaryArchSuffix >= 5.4
-	devel:liblua$secondaryArchSuffix >= 5.4
+#	devel:liblua$secondaryArchSuffix
```

Comment:
Luarocks requires at least a lua devel package, making it a hard requirement for lua5.4 breaks the use for other lua devel versions.